src/main/java/frc/trigon/robot/subsystems/turret/Turret.java

```diff
@@ -0,0 +1,61 @@
+package frc.trigon.robot.subsystems.turret;
+
+import edu.wpi.first.math.geometry.*;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.trigon.robot.RobotContainer;
+import frc.trigon.robot.utilities.AllianceUtilities;
+import org.littletonrobotics.junction.Logger;
+
+public class Turret extends SubsystemBase {
```

Comment:
Extend AbstractSubsystem (which is going to be renamed to MotorSubsystem in the future) and implement everything.